experimental/packages/exporter-logs-otlp-http/package.json

```diff
@@ -89,7 +89,7 @@
     "karma-webpack": "4.0.2",
     "mocha": "10.0.0",
     "nyc": "15.1.0",
-    "rimraf": "3.0.2",
+    "rimraf": "5.0.1",
```

Comment:
do we even need this dependency?

Reply:
Ah, this made it into the new exporter package as the code for this package was written before I removed it in #3737. It keeps coming back this way. Opened #3840 to remove it.